app/src/androidTest/java/com/github/epfl/meili/photo/PhotoEditActivityTest.kt

```diff
@@ -142,6 +142,5 @@ class PhotoEditActivityTest {
             )
         )
         fab.check(ViewAssertions.matches(isDisplayed()))
-        fab.perform(click())
```

Comment:
Why is this being removed?

Reply:
Because it is what made the tests fail on cirrus

Reply:
This should be fixed in #147, since clicking on fab now doesn't pass through CameraActivity, the intent goes directly to the caller of Camera Activity

Reply:
Perfect, thanks!